src/test/java/com/example/bank/transfer/TransferServiceTest.java

```diff
@@ -1,196 +0,0 @@
-package com.example.bank.transfer;
-
-import com.example.bank.account.AccountRepository;
-import com.example.bank.account.model.Account;
-import com.example.bank.account.model.Currency;
-import com.example.bank.client.model.Client;
-import com.example.bank.client.model.Role;
-import com.example.bank.exception.ExceptionType;
-import com.example.bank.exception.RestException;
-import com.example.bank.transfer.feign.RateClient;
-import com.example.bank.transfer.feign.RateResponse;
-import com.example.bank.transfer.feign.Rates;
-import com.example.bank.transfer.model.TransferHistory;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.MethodSource;
-import org.mockito.ArgumentCaptor;
-import org.mockito.Captor;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
-
-import java.math.BigDecimal;
-import java.security.Principal;
-import java.util.List;
-import java.util.Optional;
-
-import static org.mockito.ArgumentMatchers.anyList;
-import static org.mockito.Mockito.*;
-
-@ExtendWith(MockitoExtension.class)
-class TransferServiceTest {
```

Comment:
- @Captor → Niepotrzebne użycie listy w ArgumentCaptor
-  shouldThrowRestExceptionWhenSenderAccountNotFound() – Błędne użycie thenThrow()
- Brak testu dla przelewu na to samo konto
-  Brak testu dla negatywnej kwoty przelewu
-  Brak testu dla pustego tytułu przelewu
- Brak testu dla różnych walut, gdy użytkownik nie jest administratorem
-  shouldProcessBankTransferWhenAccountsAndBalanceAreValid() - mozesz uproscic. jak nie wiesz jak to zostaw. Tak samo jak wszedzie gdzie pisalem ze mozna uproscic. 
-